alternative_backend/boards/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 2.1.4 on 2018-12-16 22:09
+# Generated by Django 2.1.4 on 2019-01-17 11:32
```

Comment:
Nie regeneruj migracji, dodawaj kolejne migracje, które będą modyfikować projekt w trakcie jego trwania.
W tym przypadku, gdybyś już miał wersję testową uruchomioną na serwerze to musiał byś wszystko usunąć, aby zrobić aktualizację.
Warto robić kolejne migrację do łatwych aktualizacji na serwerze.